pkg/capture/capture.go

```diff
@@ -220,15 +178,13 @@ func (cmd captureCommandRotate) execute(c *Capture) stateFn {
 		stats := c.tryGetCaptureStats()
 		lastRotationStats := *stats
 
-		sub(stats, c.lastRotationStats.CaptureStats)
+		types.Sub(stats, c.lastRotationStats.CaptureStats)
```

Comment:
This (`types.Sub`) is very ambiguous. The package by now contains a ton of types, so IMHO it should be `types.SubStats()` or similar or be a method of `types.PacketStats`.
That being said: I don't think we need our own Stats type (since by now it's really just the stats from `slimcap`, so _any_ source is guaranteed to provide it via the source interface).